### PR TITLE
[137] Product Page using HTML

### PR DESCRIPTION
**Pull request by Kifaayah**
This my product page using only HTML

### CODE REVIEW

File: submissions/jimohKafayat/HTML/{137}productpage.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		<li><a href="#">Men Jalabiya</a></li>
+		<li><a href="#">Dubai Wears</a></li>
+		<li><a href="#">Women Jalabiya</a></li>
+		<li><a href="#">Ladies Gowns</a></li>
+		<li><a href="#">Corporate Wears</a></li>
+		<li><a href="#">Egyptian Gowns</a></li>
+		<li><a href="#">Veils/Hijab</a></li>
+		<li><a href="#">Islamic Kaftans</a></li>
+	</ol>
+	<img src="dufuna/images/myimage.PNG" width="400" height="500">
```
Comment (adaugochi) [inline submissions/jimohKafayat/HTML/{137}productpage.html:23]: These images used here suppose to be part of your PR. Place these images your `HTML` folder

**Review by adaugochi — CHANGES_REQUESTED** [review]
Remove the `{137}` in your `{137}productpage.html` file name

File: submissions/jimohKafayat/HTML/productpage.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		<li><a href="#">Men Jalabiya</a></li>
+		<li><a href="#">Dubai Wears</a></li>
+		<li><a href="#">Women Jalabiya</a></li>
+		<li><a href="#">Ladies Gowns</a></li>
+		<li><a href="#">Corporate Wears</a></li>
+		<li><a href="#">Egyptian Gowns</a></li>
+		<li><a href="#">Veils/Hijab</a></li>
+		<li><a href="#">Islamic Kaftans</a></li>
+	</ol>
+	<img src="dufuna/images/myimage.PNG" width="400" height="500">
```
Comment (adaugochi) [inline submissions/jimohKafayat/HTML/productpage.html:23]: This doesn't point to the location of where your images are

**Review by adaugochi — CHANGES_REQUESTED** [review]
Inside the `HTML` folder, create another folder called `images` place the two images you used for this task inside that `images` folder

**Comment by Kifaayah** [comment]
The image is in the “images” subfolder inside the dufuna folder in my IDE

**Comment by Ireti203** [comment]
Create a folder for images in the HTML folder, then change the directory in your code appropriately.

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**